ui/components/MesheryFilters/FiltersCard.js

```diff
@@ -0,0 +1,301 @@
+//@ts-check
+import React, { useState } from "react";
+import {
+  Dialog, DialogActions, DialogContent, DialogTitle, Divider, Grid, IconButton, Tooltip, Typography
+} from "@material-ui/core";
+import Button from "@material-ui/core/Button";
+import DeleteIcon from "@material-ui/icons/Delete";
+import Fullscreen from "@material-ui/icons/Fullscreen";
+import Moment from "react-moment";
+import FlipCard from "../FlipCard";
+import { makeStyles } from "@material-ui/core/styles";
+import { UnControlled as CodeMirror } from "react-codemirror2";
+import FullscreenExit from "@material-ui/icons/FullscreenExit";
+import DoneAllIcon from '@material-ui/icons/DoneAll';
+
+const INITIAL_GRID_SIZE = { xl : 4, md : 6, xs : 12 };
+
+const useStyles= makeStyles(() => ({
+  cardButtons : {
+    display : "flex",
+    justifyContent : "flex-end",
+    alignItems : "center",
+    marginTop : "50px"
+  },
+  testsButton : {
+    marginRight : "0.5rem"
+  },
+  perfResultsContainer : {
+    marginTop : "0.5rem"
+  },
+  backGrid : {
+    marginBottom : "0.25rem",
+    minHeight : "6rem",
+    position : "relative"
+  },
+  deleteButton : {
+    width : "fit-content",
+    margin : "10 0 0 auto",
+    position : "absolute",
+    right : 0,
+    bottom : 0,
+  },
+  yamlDialogTitle : {
+    display : "flex",
+    alignItems : "center"
+  },
+  yamlDialogTitleText : {
+    flexGrow : 1
+  },
+  fullScreenCodeMirror : {
+    height : '100%',
+    width : '100%',
+    '& .CodeMirror' : {
```

Comment:
A good point to think about if the two components has exact same classes and styles should that be re-used or not... I am in favor of reusing.